kapitan/inputs/kadet.py

```diff
@@ -102,34 +116,47 @@ def compile_file(self, config: KapitanInputTypeKadetConfig, input_path, compile_
         target_name = self.target_name
```

Comment:
I would suggest changing slightly the initial line of this function's docstring from:
```
Compile a kadet input file.
```
to
```
Compile a kadet input file, if resulting compilation is not already cached
```